internal/configure.go

```diff
@@ -177,6 +182,19 @@ func ConfigureLogging() error {
 	return nil
 }
 
+func getArchiveDataFolderPath() string {
+	return filepath.Join(getWalFolderPath(), "walg_archive_status")
```

Comment:
Let's move this folder to 'walg_data' folder to collect all walg data in one folder.